var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -99,6 +99,8 @@ class Podio(CMakePackage):
 
     conflicts("+rntuple", when="@:0.16", msg="rntuple support requires at least podio@0.17")
 
+    patch("fix_missing_algorithm_include.patch", when="@0.17:0.100")
+
```

Comment:
```suggestion
  patch(
      "https://github.com/AIDASoft/podio/commit/0222a077aaff817b21a46a590af0f8329dd27d67.patch?full_index=1",
      when="@0.17:0.99",
  )

```

You can use a commit that is already in the repository as the patch and the version range was wrong, with this suggestion it would mean until 0.99 including 0.99.